package.json

```diff
@@ -40,21 +40,22 @@
     "@ethereumjs/rlp": "^4.0.0",
     "@ethereumjs/tx": "^4.1.1",
     "@ethereumjs/util": "^8.0.0",
+    "@metamask/eth-sig-util": "^6.0.1",
     "@metamask/utils": "^5.0.0",
-    "eth-sig-util": "^2.0.0",
     "hdkey": "0.8.0"
   },
   "devDependencies": {
     "@ethereumjs/common": "^3.1.1",
     "@lavamoat/allow-scripts": "^2.3.0",
     "@ledgerhq/hw-app-eth": "^6.32.0",
+    "@ledgerhq/types-cryptoassets": "^7.6.0",
```

Comment:
Why are these extra dependencies necessary? I don't see them being used in this PR.